### PR TITLE
[Rector] Apply Rector: InlineIfToExplicitIfRector

### PR DESCRIPTION
**Pull request by samsonasik**
Change inline if to explicit if with `InlineIfToExplicitIfRector` rector rule.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

**Comment by samsonasik** [comment]
I included phpstan stub `ReflectionUnionType` at https://github.com/codeigniter4/CodeIgniter4/pull/4292/commits/5e71bf9329acf48ea17b0db6230cf4589c1b2f4d for handle  "Class 'ReflectionUnionType' not found" in php < 8.0

**Comment by samsonasik** [comment]
I updated github action config to use php 7.4 and 8.0 for rector check to ensure work in both php 7 and php 8

**Comment by samsonasik** [comment]
All green 🎉

File: system/Email/Email.php [hunk 1]
```diff
@@ -2367,7 +2367,10 @@ public function printDebugger($include = ['headers', 'subject', 'body'])
 
 		is_array($include) || $include = [$include]; // @phpstan-ignore-line
```
Comment (paulbalandan) [inline system/Email/Email.php:2368]: I know it is out-of-scope, but should `InlineIfToExplicitIfRector` also refactor boolean OR and concatenations? Looking at the code it only deals with `BooleanAnd` and `Assign` types.

File: system/Email/Email.php [hunk 2]
```diff
@@ -2367,7 +2367,10 @@ public function printDebugger($include = ['headers', 'subject', 'body'])
...
 
-		in_array('headers', $include, true) && $rawData  = htmlspecialchars($this->headerStr) . "\n";
+		if (in_array('headers', $include, true))
+		{
+			$rawData = htmlspecialchars($this->headerStr) . "\n";
+		}
 		in_array('subject', $include, true) && $rawData .= htmlspecialchars($this->subject) . "\n";
 		in_array('body', $include, true) && $rawData    .= htmlspecialchars($this->finalBody);
```
Comment (paulbalandan) [inline system/Email/Email.php:2375]: If we could have the Rector rule cover these two also that would be great.

**Review by paulbalandan — APPROVED** [review]
I approve, but I have some comments that may be better addressed at `rectorphp/rector` for improvement of the rector rule.

File: system/Email/Email.php (hunk 1 above)
Comment (samsonasik) [reply system/Email/Email.php:2368]: Yes, BooleanOr should be covered, that maybe next improvement 👍

File: system/Email/Email.php (hunk 2 above)
Comment (samsonasik) [reply system/Email/Email.php:2375]: Yes, hopefully next improvement 👍

File: system/Email/Email.php (hunk 1 above)
Comment (samsonasik) [reply system/Email/Email.php:2368]: implemented in rector dev-master, wait for next release ;)

File: system/Email/Email.php (hunk 2 above)
Comment (samsonasik) [reply system/Email/Email.php:2375]: implemented in rector dev-master, wait for next release ;)

**Review by MGatner — COMMENTED** [review]
I'm a fan of this change!

**Pull request merged**